test_util/integration_test.py

```diff
@@ -15,13 +18,19 @@
 import requests
 import retrying
 
+from ssh.ssh_tunnel import SSHTunnel, run_ssh_cmd
```

Comment:
This import might break, depending on how @mellenburg is "installing" the libraries via the Dockerfile. The last I recall from my convo with Michael is that the import should be `import ssh_tunnel` rather than `from ssh import ssh_tunnel`; by extension, `from ssh.ssh_tunnel import foo` may not work. This will work locally, but not as the tests are intended to be run.